src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: block;
+}
+
+/* Блок зірок */
+.stars {
+  display: flex;
+}
+
+div {
+  unicode-bidi: isolate;
+}
```

Comment:
Avoid using tag names for styling (except `html` and `body`). It's recommended to style elements using `.classes`. Replace the `div` selector with a class selector that targets the specific elements you wish to style.